gammapy/data/event_list.py

```diff
@@ -232,9 +244,54 @@ def select_sky_cone(self, center, radius):
         """
         position = self.radec
         separation = center.separation(position)
-        mask = separation > radius
+        mask = separation < radius
         return self[mask]
 
+    def select_sky_ring(self, center, inner_radius = None, outer_radius = None,
```

Comment:
My suggestion would be to do a simpler API / implementation here:
```
def select_sky_ring(self, center, inner_radius, outer_radius)
```
If the user has a thickness, it's still short and readable:
```
select_sky_ring(center, inner_radius, inner_radius + thickness)
```
But if you like the flexibility here, OK to keep (the implementation with the many if statements looks correct to me).